toolkit/offchain/src/csl.rs

```diff
@@ -288,7 +288,9 @@ impl TransactionBuilderExt for TransactionBuilder {
 		inputs: &Vec<OgmiosUtxo>,
 	) -> Result<(), JsError> {
 		let mut collateral_builder = TxInputsBuilder::new();
-		collateral_builder.add_key_inputs(&inputs, &ctx.payment_key_hash())?;
+		for utxo in inputs.iter() {
+			collateral_builder.add_regular_input(&key_hash_address(&ctx.payment_key_hash(), ctx.network), &utxo.to_csl_tx_input(), &convert_value(&utxo.value)?)?;
+		}
```

Comment:
`add_key_inputs` were adding required witness. I don't know if it is needed. I've looked at CSL examples and for collaterals they were using `add_regular_input`. Perhaps has little significance for us, since we add payment vkey witness anyway.